doc/release-notes-27307.md

```diff
@@ -0,0 +1,8 @@
+Wallet
+---
+
+The wallet now detects when wallet transactions conflict with the mempool. Mempool
+conflicting transactions can be seen in the `"mempoolconflicts"` field of
+`gettransaction`. The inputs of mempool conflicted transactions can now be respent
+without manually abandoning the transactions, which can cause wallet balances to
+appear higher.
```

Comment:
can now be respent without manually abandoning the transactions "when the parent transaction is dropped from the mempool"?

Reply:
Done